src/components/chips/js/chipsDirective.js

```diff
@@ -194,6 +194,7 @@
       scope: {
         readonly: '=readonly',
         placeholder: '@',
+        enableChipEdit: '@',
```

Comment:
Ah just rechecked the code and noticed the attribute name. Whenever we provide custom attributes, we should prefix them with `md-` like `md-enable-chip-edit`